File: Java/Program/Ram.java
public class Ram extends Komponen {
    private int kapasitasGB;
    private String ddr;

    public Ram(String merk, String nama, int kapasitasGB, String ddr) {
        super(merk, nama);
        this.kapasitasGB = kapasitasGB;
        this.ddr = ddr;
    }

    public void setKapasitasGB(int kapasitasGB) {
        this.kapasitasGB = kapasitasGB;
    }

    public int getKapasitasGB() {
        return kapasitasGB;
    }

    public void setDdr(String ddr) {
        this.ddr = ddr;
    }

    public String getDdr() {
        return ddr;
    }
}

File: Python/Program/Ram.py
from Komponen import Komponen

class Ram(Komponen):
    def __init__(self, merk, nama, kapasitasGB, ddr):
        super().__init__(merk, nama)
        self.kapasitasGB = kapasitasGB
        self.ddr = ddr
        
    def set_kapasitasGB(self, kapasitasGB):
        self._kapasitasGB = kapasitasGB
        
    def get_kapasitasGB(self):
        return self._kapasitasGB
    
    def set_ddr(self, ddr):
        self._ddr = ddr
        
    def get_ddr(self):
        return self._ddr